src/main/java/cz/xtf/TestConfiguration.java

```diff
@@ -58,6 +58,8 @@
 	public static final String CI_USERNAME = "ci.username";
 	public static final String CI_PASSWORD = "ci.password";
 
+	public static final String OC_BINARY_LOCATION = "oc.binary.location";
```

Comment:
OC_BINARY_LOCATION with dependent methods should be in XTFConfiguration class, it is general purpose property

Reply:
fixed